src/main/java/com/google/api/generator/engine/ast/OperationExpr.java

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.ast;
+
+public interface OperationExpr extends Expr {
+
+  OperatorKind operatorKind();
```

Comment:
Could we add `type()` here as well? Although this is technically fine, adding `type()` here will help us see all the methods that need to be implemented in subclasses.

Reply:
sg

Reply:
Since our purpose is to make all methods need to be implemented visible, I also added `accept`. LMK if you have other opinion. :)

Reply:
Yes, thanks for doing that.